subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/CacheInitScriptsSmokeTest.groovy

```diff
@@ -22,27 +22,18 @@ class CacheInitScriptsSmokeTest extends AbstractSmokeTest {
 
     def 'cache init script plain text output'() {
```

Comment:
Looks like we can remove the whole smoke test now, given that is was only used to test the cache init scripts - which this PR removes.

Reply:
It still tests that the cache can be initialized from an init script. Do you want to remove that as well?

Reply:
Let's remove it, we already have a test for that: https://github.com/gradle/gradle/blob/06677b922432b742db3b81fb3938616f83b82387/subprojects/core/src/integTest/groovy/org/gradle/caching/configuration/internal/BuildCacheConfigurationIntegrationTest.groovy#L79-L96

Testing that the cache can be initialized from an init script should not be a smoke test.

Reply:
Removed.